core/jobs/batch_jobs/suggestion_edit_state_content_deletion_jobs_test.py

```diff
@@ -0,0 +1,286 @@
+            target_id=target_id,
+            target_version_at_submission=1,
+            status='accepted',
+            author_id=author_id,
+            final_reviewer_id=final_reviewer_id,
+            change_cmd={},
+            score_category=(
+                suggestion_models.SCORE_TYPE_TRANSLATION +
+                suggestion_models.SCORE_CATEGORY_DELIMITER + 'English'),
+            language_code=language_code
+        )
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.edit_state_content_suggestion_model_1: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_1',
+                'author_1',
+                'reviewer_1',
+                None
+            ))
+
+        self.edit_state_content_suggestion_model_2: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_2',
+                'author_2',
+                'reviewer_2',
+                None
+            ))
+
+        self.create_translation_suggestion_model: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_3',
+                'author_3',
+                'reviewer_3',
+                'hi'
+            ))
+
+        self.add_question_suggestion_model: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_ADD_QUESTION,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_4',
+                'author_4',
+                'reviewer_4',
+                'en'
+            ))
+
+    def test_job_deletes_suggestion_edit_state_content_model(self) -> None:
+        self.edit_state_content_suggestion_model_1.update_timestamps()
+        self.edit_state_content_suggestion_model_2.update_timestamps()
+        self.create_translation_suggestion_model.update_timestamps()
+        self.add_question_suggestion_model.update_timestamps()
+        suggestion_model.put_multi([
+            self.edit_state_content_suggestion_model_1,
+            self.edit_state_content_suggestion_model_2,
+            self.create_translation_suggestion_model,
+            self.add_question_suggestion_model])
+
+        queries = [(
+            'suggestion_type',
+            feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT)]
+
+        self.assertEqual(
+            len(suggestion_model.query_suggestions(
+                queries)), 2)
+
+        self.assert_job_output_is([
+            job_run_result.JobRunResult(
+                stdout='EDIT STATE CONTENT SUGGESTION SUCCESS: 2')
+        ])
+
+        self.assertEqual(
+            len(suggestion_model.query_suggestions(
+                queries)), 2)
```

Comment:
You should be able to move this to the end of the previous line?

Reply:
done